content/en/infrastructure/serverless/datadog_lambda_layer.md

```diff
@@ -0,0 +1,184 @@
+---
+title: Datadog Lambda Layer
+kind: documentation
+aliases:
+  - infrastructure/serverless/datadog-lambda-layer
+  - infrastructure/serverless/lambda_layer
+  - infrastructure/serverless/lambda-layer
+further_reading:
+- link: "/integrations/amazon_lambda/"
+  tag: "AWS Lambda Integration"
+  text: "AWS Lambda Integration"
+---
+
+The Datadog Lambda Layer is responsible for:
+
+- Generating real-time [enhanced Lambda metrics][1] for invocations, errors, cold starts, etc.
+- Submitting custom metrics (synchronously and asynchronously)
+- Automatically propagating tracing headers from upstream requests to downstream services. This enables full distributed tracing across Lambda functions, hosts, containers, and other infrastructure running the Datadog Agent.
```

Comment:
(Below this) Let's also update this bullet list to say that it packages the `dd-trace` library, letting customers trace their Lambda functions with Datadog's tracing libraries. currently available for Node.js, Python, and Ruby with more runtimes coming.

Reply:
fixed